Library/MessageGenerator.swift

```diff
@@ -198,11 +198,7 @@ struct MessageGenerator {
     }
 
     func minuteString(for interval: Double) -> String {
-        let minutes = interval / 60
-
-        let formatter = NumberFormatter()
-        formatter.maximumFractionDigits = 2
-        let minutesString = formatter.string(from: NSNumber(value: abs(minutes)))!
+        let minutesString = String(describing: Int(round(interval / 60)))
```

Comment:
I was actually working on writing a test for this. Might be good to do now? 😬

I don't really understand why you use `describing`?